PYME/recipes/surface_fitting.py

```diff
@@ -201,4 +201,31 @@ def execute(self, namespace):
             #target_length = np.mean(surf._halfedges['length'][surf._halfedges['length'] != -1])
             surf.remesh(5, l=0.5, n_relax=10)
         
-        namespace[self.output] = surf
\ No newline at end of file
+        namespace[self.output] = surf
+
+@register_module('DistanceToSurface')
+class DistanceToSurface(ModuleBase):
```

Comment:
I wonder if it might not be better to call this `DistanceToMesh` to make it a bit clearer what the input should be. There are multiple ways of defining/parameterising a surface, with a mesh being only one of them.

Reply:
Addressed